qa/common/cuda_op_kernel.cu.cc.patch

```diff
@@ -7,7 +7,7 @@ index a9d66f9..a92e218 100644
 
  #if GOOGLE_CUDA
 -#define EIGEN_USE_GPU
--#include "third_party/eigen3/unsupported/Eigen/CXX11/Tensor"
+-#include "unsupported/Eigen/CXX11/Tensor"  // from @eigen_archive
 -#include "tensorflow/core/util/gpu_kernel_helper.h"
 -#include "tensorflow/core/util/gpu_launch_config.h"
 +//#define EIGEN_USE_GPU
```

Comment:
Why is this getting taken out?

Reply:
There was a path change within the TF ontainer